packages/coding-style/src/Rector/ClassMethod/AnnotateThrowables.php

```diff
@@ -98,11 +120,12 @@ private function isDocCommentAlreadyPresent(Doc $docComment, string $pattern):bo
     /**
      * @param string $FQNOrThrowableName
      *
-     * @return string
+     * @return AttributeAwarePhpDocTagNode
      */
-    private function buildThrowsDocComment(string $FQNOrThrowableName):string
+    private function buildThrowsDocComment(string $FQNOrThrowableName):AttributeAwarePhpDocTagNode
     {
-        return sprintf('@throws %s', $FQNOrThrowableName);
+        $value = new AttributeAwareGenericTagValueNode($FQNOrThrowableName);
```

Comment:
This should be probably `AttributeAwareIdentifierTypeNode`

Reply:
I don't know the difference, honestly 😅
I'll change it as you suggest.

Reply:
`AttributeAwareIdentifierTypeNode` seems to not be compatible with `AttributeAwarePhpDocTagNode`.

Can you tell me what should have I to use?